libr/util/buf.c

```diff
@@ -65,6 +68,12 @@ static ut8 *get_whole_buf(RBuffer *b, ut64 *sz) {
 	if (bsz == UT64_MAX) {
```

Comment:
could you remove also the comment before this line as well since you are at it? i don't think it makes any sense :/